### ISSUE TITLE
Remove Namespaces From UI

### ISSUE THREAD

**Issue by gitsmith** [domaindrivendev/Swashbuckle#631]
We've got a few classes with the same name in our APIs and needed to [UseFullTypeNamesInSchemaIds](https://github.com/domaindrivendev/Swashbuckle#usefulltypenamesinschemaids).

We'd prefer not to display model namespaces. Is there any way we can customize the UI as to hide and/or remove the namespace from the model definition?

**Comment by domaindrivendev** [comment]
2 potential approaches ...

1) Provide you're own strategy for emitting unique SchemaId's (perhaps that don't include the complete namespace) server side via the SchemaId config action - https://github.com/domaindrivendev/Swashbuckle#schemaid
2) Inject some custom JavaScript that just hides the namespace in the UI (see readme)

**Comment by gitsmith** [comment]
Thanks for the feedback. 

I had implemented both of these options and each came with their own trade-offs.

1) Dynamically generating schema ids doesn't result in overly friendly class names. 

2) This was a pretty simple option which works well if you are only working with the enabled swagger UI. Unfortunately, at the moment, we also have a need to create a second custom document so we modify the raw swagger that is generated. Of course, since this option is JavaScript the classes include the fully qualified namespaces.

Ended up renaming the classes.
